cli/root.go

```diff
@@ -34,6 +35,10 @@ var RootCommand = &cobra.Command{
 			"version",
 		}
 
+		if !git.IsValidTagName(strings.TrimSuffix(rootPrefixFlag, ".")) {
```

Comment:
Shouldn't the trailing dot be handled by the `IsValidTagName` function, so that the call to `TrimSuffix` can be removed here?

Reply:
In this case, I try to address a case where the prefix should be something like `usa.` (resulting in `usa.1.2.3`). The prefix on its own would not be a valid tag name. Due to the dot is not at the end of the version, the resulting one should be valid.

While I am writing this comment, I see that solving that use case is not even as simple as I thought (trailing slashes/dots, ...).

Will be rewritten soon.